components/login/login.tsx

```diff
@@ -659,8 +659,8 @@ const Login = ({onCustomizeHeader}: LoginProps) => {
     };
 
     const getMessageSubtitle = () => {
-        if (enableCustomBrand && CustomBrandText) {
-            return (
+        if (enableCustomBrand) {
+            return CustomBrandText && (
```

Comment:
Curious why this change? Seems to be have the same result as before?

Reply:
Before it only entered here if both Custom Brand was enabled and Text was provided. Else the default text was always returned.
Now only having Custom Brand enabled it enters here, and no default text is returned.